internal/controller/syncmanager/controller.go

```diff
@@ -161,9 +178,9 @@ func Add(
 	return err
 }
 
-func (r *Reconciler) Reconcile(ctx context.Context, _ reconcile.Request) (reconcile.Result, error) {
+func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
 	log := r.log.Named(ControllerName)
-	log.Debug("Processing")
+	log.Debugw("Processing", "request", req)
```

Comment:
This controller only ever reconciles a dummy object, it has no meaning to log this dummy value.